cups/ppd-cache.c

```diff
@@ -891,6 +893,28 @@ _ppdCacheCreateWithFile(
           cupsParseOptions(valueptr, 0,
 	                   pc->presets[print_color_mode] + print_quality);
     }
+    else if (!_cups_strcasecmp(line, "OptimizePreset"))
```

Comment:
Not a preset, don't use the "preset" name for it!

Reply:
I could name it differently.